core/include/userver/server/handlers/http_handler_base.hpp

```diff
@@ -190,6 +190,7 @@ class HttpHandlerBase : public HandlerBase {
   bool set_response_server_hostname_;
   mutable utils::TokenBucket rate_limit_;
   bool is_body_streamed_;
+  bool tracing_headers_enabled_;
```

Comment:
Please add to `server::handlers::HandlerConfig` instead

Reply:
I thought about disabling these headers on per-handler basis, but thinking more about this i tend to believe this is indead too granular. Will move into common config

Reply:
I decided it wouldn't hurt to have a possibility to set this at a server level and an ability to override at per-handler level